cli/src/commands/branch.rs

```diff
@@ -301,6 +317,44 @@ fn cmd_branch_create(
     Ok(())
 }
 
+fn cmd_branch_rename(
+    ui: &mut Ui,
+    command: &CommandHelper,
+    args: &BranchRenameArgs,
+) -> Result<(), CommandError> {
+    let mut workspace_command = command.workspace_helper(ui)?;
+    let repo = workspace_command.repo().as_ref();
+    let view = repo.view();
+    let old_branch = &args.old;
+    // call clone because .set_local_branch_target takes ownership
+    // of the target_commit, so we need to create a copy that is no
+    // longer owned by the workspace.
```

Comment:
nit: `.clone()` is very common throughout the code base (and in Rust code in general, probably). I don't think it's worth mentioning why we need to clone here.

Reply:
Makes sense. Removed.